AstronomyUITests/AstronomyUITests.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AstronomyUITests.swift
+//  AstronomyUITests
+//
+//  Created by Cora Jacobson on 9/24/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+
+class AstronomyUITests: XCTestCase {
+    
+    let app = XCUIApplication()
+    
+    private var nextSolButton: XCUIElement {
+        return app.buttons["PhotosCollectionViewController.NextSolButton"]
+    }
+    
+    private var previousSolButton: XCUIElement {
+        return app.buttons["PhotosCollectionViewController.PreviousSolButton"]
+    }
+    
+    private var savePhotoButton: XCUIElement {
+        return app.buttons["PhotoDetailViewController.SaveButton"]
+    }
+    
+    private var firstChild: XCUIElement {
+        return app.collectionViews.children(matching: .cell).element(boundBy: 0)
+    }
```

Comment:
Very nice! Using computed properties to get the buttons and cell and giving them nice names. I would argue that `firstChild` might be better named `firstCell`, but that's just a nitpick, and this is very thoughtful of you to do this and keep your tests clean!